src/components/theme/getThemeValue.ts

```diff
@@ -0,0 +1,5 @@
+import {getSystemTheme} from './getSystemTheme';
```

Comment:
What is the point of extracting this into single file if the only place it used is ThemeProvider? Same question for getSystemTheme

Reply:
Small modules is easy to refactor, you don't have to move them from one file to another
Changes to small module is easier to code review, because it scoped to module with single responsibility
It is easier to rebase smaller modules